server/src/main/java/com/distkv/server/storeserver/StoreServer.java

```diff
@@ -1,17 +1,23 @@
 package com.distkv.server.storeserver;
 
 import com.distkv.rpc.service.DistkvService;
+import com.distkv.server.AbstractCompositeService;
+import com.distkv.server.Address;
+import com.distkv.server.Node;
 import com.distkv.server.storeserver.runtime.StoreRuntime;
 import com.distkv.server.storeserver.services.DistkvServiceImpl;
 import org.dousi.DousiServer;
 import org.dousi.config.ServerConfig;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class StoreServer {
+public class StoreServer extends AbstractCompositeService {
```

Comment:
The StoreServer here is just a starter. We put all logic to StoreRuntime.
How about make StoreRuntime extends from AbstractCompositeService? 
And put the node info to Runtime?

Reply:
yes, you are right.  
I will make StoreRuntime as StoreService at another pr. because this pr is a little large.

Reply:
But this PR shouldn't be checked in master since it's not reasonable make StoreServer as a Service.

How about finishing another one to make Runtime extends from service and then we check in this PR?

Reply:
Ok, I will make runtime as a servcie at this pr. because the runtime denpendent on the service interface at this pr.